common/data_refinery_common/models/models.py

```diff
@@ -324,7 +324,7 @@ def save(self, *args, **kwargs):
         return super(ExperimentAnnotation, self).save(*args, **kwargs)
 
 class Pipeline(models.Model):
-    "Pipeline that is associated with a series of ComputationalResult records."""
+    """Pipeline that is associated with a series of ComputationalResult records."""
```

Comment:
:scream: What was happening before this?!?

Reply:
Fortunately, it was just executing the string `"Pipeline that is associated with a series of ComputationalResult records."` and then the empty string `""`.

Reply:
Oh whew!